core/util/hyperparameter_baseline.py

```diff
@@ -0,0 +1,13 @@
+from core.util.loss_functions import zap_loss
+
+
```

Comment:
Could change this to hyperparameter_configuration, as baseline does not seem to convey the meaning. Also small typo in "hyperparamter" -> "hyperparameter"